src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/EditJudgmentCallbackHandler.java

```diff
@@ -98,14 +98,17 @@ private CallbackResponse buildConfirmation(CallbackParams callbackParams) {
     private CallbackResponse saveJudgmentDetails(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
         List<String> errors = new ArrayList<>();
+
         if (caseData.getJoIsRegisteredWithRTL() == YesOrNo.YES) {
             caseData.setJoIssuedDate(caseData.getJoOrderMadeDate());
         }
         if (caseData.getActiveJudgment() != null) {
             caseData.setActiveJudgment(editJudgmentOnlineMapper.addUpdateActiveJudgment(caseData));
+            caseData.setJoRepaymentSummaryObject(JudgmentsOnlineHelper.calculateRepaymentBreakdownSummary(caseData.getActiveJudgment()));
```

Comment:
Should it be changed to caseDataBuilder here too?

Reply:
Done.